web/app/themes/ppj/src/vue/FindAJob.vue

```diff
@@ -163,13 +171,16 @@
           <button class="find-a-job__pagination-direction"
                   :class="{'find-a-job__pagination-direction--enabled': (forwardEnabled == true)}"
                   @click.stop.prevent="showNextPage">
-            <svg x="0px"
-                 y="0px"
-                 viewBox="0 0 32.9 32.9"
+            <svg
+              aria-labelledby="nextPageButtonTitle nextPageButtonDesc"
+              viewBox="0 0 32.9 32.9"
+              x="0px"
+              y="0px"
             >
-                  <path class="st0" d="M11.7,21.2v-0.9c0-0.2,0.1-0.3,0.2-0.3l7.5-3.4c0.1,0,0.1-0.1,0-0.1L11.9,13c-0.1-0.1-0.2-0.2-0.2-0.3v-0.9
-                    c0-0.2,0.1-0.3,0.3-0.2l8.9,4.1c0.1,0.1,0.2,0.2,0.2,0.4v0.9c0,0.2-0.1,0.3-0.2,0.4l-8.9,4.1C11.9,21.5,11.7,21.4,11.7,21.2z"/>
-
+              <title id="nextPageButtonTitle">Next page button</title>
+              <desc id="nextPageButtonDesc">A shape like a greater than sign</desc>
```

Comment:
"An arrow pointing right" might be a more user friendly description of this image.

Reply:
Will discuss with Josh.